src/stores/useToolStore/index.ts

```diff
@@ -135,9 +134,19 @@ export const useToolStore = create<ToolState>()((set) => ({
         ehdotukset,
       );
 
-      const sortedMixedMahdollisuudet = [...[...koulutusmahdollisuusResults, ...tyomahdollisuusResults]].sort((a, b) =>
-        ehdotukset ? (ehdotukset[b.id]?.pisteet ?? 0) - (ehdotukset[a.id]?.pisteet ?? 0) : 0,
-      );
+      const typedTyomahdollisuusResults = tyomahdollisuusResults.map((km) => ({
+        ...km,
```

Comment:
Not that it actually matters, but maybe `km` could be `tm` to match with the pattern.

Reply:
Changed